### DISCUSSION TITLE
When contributing, pulls are tagged as gB Open - how do I tag them as gB Humanitarian?

### DISCUSSION THREAD

**Discussion by MichaelBrandonFalk** [wmgeolab/geoBoundaries#2342] [General]
When I upload new shapefiles to geoBoundaries via the website tool https://www.geoboundaries.org/gbContribute.html
it by default tags the pull as gB Open. Will there be the option to tag it as gB humanitarian at some point?

**Comment by DanRunfola** [comment]
Tentatively I think the answer might be no, at least through that interface, as the gbHumanitarian (and Authoritative) are really just mirrors of UN SALB and UN OCHA.  What I hope to do in the future (this year) is rebrand those so that's clear -- i.e., get rid of the words "gbHumanitarian" and "gbAuthoritative", which are artifacts of older definitions of those datasets.
I think the "right" answer is going to be modifying the API you're using a bit to allow for programmatic choices of other cases.  In the interim, would it be possible for you to flag "Humanitarian" in one of the other fields (if you aren't already)?

**Reply by MichaelBrandonFalk** [reply]
Will do! Thanks Dan.  @leeberryman  tagging you here as I believe you were interested in this as well.

**Comment by DanRunfola** [comment]
Closing this as the discussion has continued in https://github.com/wmgeolab/geoBoundaries/issues/2295

**Reply by DanRunfola** [reply]
Err - well, not closing it, but rather flagging that the discussion has continued in an issue.  Still new to these github discussion boards :)

**Reply by MichaelBrandonFalk** [reply]
Sounds good :) Thanks Dan.
